code/modules/projectiles/projectile/hookshot.dm

```diff
@@ -174,3 +180,42 @@
 
 /obj/item/projectile/hookshot/ex_act()
 	return
+
+//Whips
+
+/obj/item/projectile/hookshot/whip
+	name = "whip"
+	icon_state = "whip"
+	icon_name = "whip"
+	fire_action = "flick"
+	nodamage = 0
+	damage = 15
+	kill_count = 5
+	sharpness = 1.2
+	failure_message = "Your hand slips and the whip falls loose..."
+	can_tether = FALSE
+	var/whipitgood_bonus = 5
+
+/obj/item/projectile/hookshot/whip/on_hit(var/atom/atarget, var/blocked = 0)
+	var/obj/item/weapon/gun/hookshot/whip/W = shot_from
+	if(W.firer?.is_wearing_item(/obj/item/clothing/head/energy_dome) && whipitgood_bonus)
+		force += whipitgood_bonus
+		visible_message("<span class='warning'>[W.firer] whips it good!</span>")
+	..(atarget, blocked)
+
+/obj/item/projectile/hookshot/whip/vampkiller
+	name = "flail"
+	icon_state = "vampkiller"
+	icon_name = "vampkiller"
+	damage = 1
```

Comment:
Doesn't this make the flail deal 1 damage per hit?

Reply:
Yes, that's intentional since this version of the item is easily obtainable from the pietyvend

Reply:
Shrug